examples/tutorial/tutorial_notebooks/introduction/omnisci_on_native/exercise_3.ipynb

```diff
@@ -0,0 +1,313 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![LOGO](../img/MODIN_ver2_hrz.png)\n",
+    "\n",
+    "<center><h2>Scale your pandas workflows by changing one line of code</h2>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 3: Not Implemented\n",
+    "\n",
+    "**GOAL**: Learn what happens when a function is not yet supported in Modin as well as how to extend Modin's functionality using the DataFrame Algebra."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When functionality has not yet been implemented, we default to pandas\n",
+    "\n",
+    "![](../img/convert_to_pandas.png)\n",
+    "\n",
+    "We convert a Modin dataframe to pandas to do the operation, then convert it back once it is finished. These operations will have a high overhead due to the communication involved and will take longer than pandas.\n",
+    "\n",
+    "When this is happening, a warning will be given to the user to inform them that this operation will take longer than usual. For example, `DataFrame.mask` is not yet implemented. In this case, when a user tries to use it, they will see this warning:\n",
+    "\n",
+    "```\n",
+    "UserWarning: `DataFrame.mask` defaulting to pandas implementation.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Concept for exercise: Default to pandas\n",
+    "\n",
+    "In this section of the exercise we will see first-hand how the runtime is affected by operations that are not implemented."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import modin.pandas as pd\n",
+    "import pandas\n",
+    "import numpy as np\n",
+    "import time\n",
+    "import modin.config as cfg\n",
+    "cfg.StorageFormat.put(\"omnisci\")\n",
+    "\n",
+    "frame_data = np.random.randint(0, 100, size=(2**18, 2**8))\n",
+    "df = pd.DataFrame(frame_data).add_prefix(\"col\")"
```

Comment:
I don't see any operation over Modin DataFrame that defaults to pandas. Should be added

Reply:
@YarShev I added this